src/components/pages/Create.vue

```diff
@@ -17,6 +19,10 @@
             </div>
             <label for="description">Description</label>
             <textarea name="description" id="description" v-model="garment.description"></textarea>
+            <label for="licence">Licence</label>
+            <select name="licence" id="licence" v-model='garment.licence'>
+                <option v-for="licence in licences" :value="licence">{{licence}}</option>
```

Comment:
By setting the default value of `garment.licence`, the option corresponding to that licence will be `selected` by default.